storcli.py

```diff
@@ -155,6 +155,11 @@
         "MegaRAID physical drive emergency spare",
         ["controller", "enclosure", "slot"], namespace=namespace, registry=registry,
     ),
+    "pd_temperature": Gauge(
+        "pd_temperature",
```

Comment:
Metrics which express a quantity should include the unit in the name. See https://prometheus.io/docs/practices/naming/#metric-names for guidance. I would suggest `pd_temp_celsius` to keep it concise but clear.

Reply:
Good to know, thanks @dswarbrick!